src/util/get-github-client-config.ts

```diff
@@ -65,7 +67,64 @@ async function calculateProxyBaseUrl(jiraHost: string, gitHubBaseUrl: string | u
 	return envVars.PROXY;
 }
 
-export const getGitHubClientConfigFromAppId = async (gitHubAppId: number | undefined, logger: Logger, jiraHost: string): Promise<GitHubClientConfig> => {
+async function buildGitHubServerConfig(githubServerBaseUrl: string, jiraHost: string, logger: Logger): Promise<GitHubConfig> {
```

Comment:
The diff looks big, however this is just splitting up the old super-method smaller ones.